pkg/controller/volume/attachdetach/populator/desired_state_of_world_populator.go

```diff
@@ -113,3 +126,21 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		dswp.desiredStateOfWorld.DeletePod(dswPodUID, dswPodToAdd.VolumeName, dswPodToAdd.NodeName)
 	}
 }
+
+func (dswp *desiredStateOfWorldPopulator) findAndAddActivePods() {
+	pods, err := dswp.podLister.List(labels.Everything())
```

Comment:
Since we are allowed to set custom resync periods when using shared informers now, would it be better to set a 30sec-1min resync period and let the existing podUpdate flow handle this? See this comment by @saad-ali https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/attachdetach/attach_detach_controller.go#L83 (which foresaw https://github.com/kubernetes/kubernetes/issues/43300, kind of)